contributing.md

```diff
@@ -0,0 +1,188 @@
+
+- Open the [Jira issue tracker](https://jira.percona.com/projects/PXC/issues) for the doc project.
+- Sign in (create a Jira account if you don’t have one) and click **Create** to create an issue.
+- In the following fields, describe the issue:
+    - In the Summary, provide a brief description of the issue
+    - In the Description, provide more information about the issue, along with a Steps To Reproduce section, if needed
+    - In the Affects Version/s field, if you know this issue affects multiple versions, please enter the version numbers. It is OK to add the version number with an ".x" (such as ``8.0.x``) if you don't know the exact version that must be updated.
+
+## Contribute to documentation yourself
+
+There is the **Edit this page** link that leads you to the source file of the page on GitHub. There you make changes, create a pull request that we review and add to the doc project. For details how to do it, read on.
+
+To contribute to documentation, learn about the following:
+
+- [Markdown](https://www.mkdocs.org/) 
+- [MkDocs](https://daringfireball.net/projects/markdown/) 
+- [git](https://git-scm.com/) and [GitHub](https://guides.github.com/activities/hello-world/)
+- [Docker](https://docs.docker.com/get-docker/). It allows you to run MkDocs in a virtual environment instead of installing it and its dependencies on your machine.
+
+There are several active versions of the documentation. Each version has a branch in the repository named accordingly:
+
+- 5.7
+- 8.0
+
+The `.md` files are in the ``docs/`` directory. 
+
+### Edit documentation online using GitHub
+
+1. Click the **pencil** icon next to the page title to edit the document. The source ``.md`` file of the page opens in GitHub editor in your browser. If you haven’t worked with the repository before, GitHub creates a [fork](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo) of it for you.
+
+2. Edit the page. You can check your changes on the **Preview** tab.
+
+3. Commit your changes.
+
+	 - In the *Commit changes* section, describe your changes.
+	 - Select the **Create a new branch for this commit and start a pull request** option
+	 - Click **Propose changes**.
+
+4. GitHub creates a branch and a commit for your changes. It loads a new page on which you can submit a pull request to Percona. The page shows the base branch - the one you offer your changes for, your commit message and a diff - a visual representation of your changes against the original page. At this time you can make a last-minute review. When you are ready, click the **Create pull request** button.
+5. Someone from our team reviews the pull request. They may make a comment or they may approve it. The approved pull request is merged into documentation when the next version of Percona XtraBackup is released. 
```

Comment:
We merge the approved pull request. This merge usually happens when Percona XtraDB Cluster is released.

Reply:
Rewritten